xmppserver/src/main/webapp/system-admin-console-access.jsp

```diff
@@ -0,0 +1,173 @@
+    final String allowedIPs = request.getParameter("allowedIPs");
+    final boolean ignoreExcludes= ParamUtils.getBooleanParameter( request, "ignore-excludes" );
+
+    boolean save = ParamUtils.getParameter(request, "save") != null;
+    boolean success = ParamUtils.getBooleanParameter(request, "success");
+
+    // Handle a save
+    Cookie csrfCookie = CookieUtils.getCookie(request, "csrf");
+    String csrfParam = ParamUtils.getParameter(request, "csrf");
+
+    if (save) {
+        if (csrfCookie == null || csrfParam == null || !csrfCookie.getValue().equals(csrfParam)) {
+            save = false;
+            errors.put("csrf", "CSRF Failure!");
+        }
+    }
+    csrfParam = StringUtils.randomString(15);
+    CookieUtils.setCookie(request, response, "csrf", csrfParam, -1);
+    pageContext.setAttribute("csrf", csrfParam);
+
+    if (save) {
+        // do validation
+        final Pattern pattern = Pattern.compile("(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\\." +
+            "(?:(?:\\*|25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\\.){2}" +
+            "(?:\\*|25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)");
+
+        final Set<String> blockedSet = new HashSet<>();
+        for (final String address : blockedIPs.split("[,\\s]+") ) {
+            if (!address.isEmpty()) {
+                if (pattern.matcher(address).matches()) {
+                    blockedSet.add(address);
+                } else {
+                    if (errors.containsKey("invalid-blocklist-ips")) {
+                        errors.put("invalid-blocklist-ips", errors.get("invalid-blocklist-ips") + ", " + address);
+                    } else {
+                        errors.put("invalid-blocklist-ips", address);
+                    }
+                }
+            }
+        }
+        final Set<String> allowedSet = new HashSet<>();
+        for (final String address : allowedIPs.split("[,\\s]+") ) {
+            if (!address.isEmpty()) {
+                if (pattern.matcher(address).matches()) {
+                    allowedSet.add(address);
+                } else {
+                    if (errors.containsKey("invalid-allowlist-ips")) {
+                        errors.put("invalid-allowlist-ips", errors.get("invalid-allowlist-ips") + ", " + address);
+                    } else {
+                        errors.put("invalid-allowlist-ips", address);
+                    }
+                }
+            }
+        }
+
+        if (errors.isEmpty()) {
+            AuthCheckFilter.IP_ACCESS_BLOCKLIST.setValue(blockedSet);
+            AuthCheckFilter.IP_ACCESS_ALLOWLIST.setValue(allowedSet);
+            AuthCheckFilter.IP_ACCESS_IGNORE_EXCLUDES.setValue(ignoreExcludes);
+
+            // Log the event
+            webManager.logEvent("Updated Admin Console access configuration.", "Blocklist = " + String.join(", ", blockedSet) +"\nAllowlist = " + String.join(", ", allowedSet)+"\nignore excludes = "+ignoreExcludes);
```

Comment:
Newlines don't show in the Audit Viewer. Semicolons instead?

![image](https://user-images.githubusercontent.com/2117083/182952058-a6f76ce1-0d6c-49e8-b888-b9685256b65a.png)

Reply:
Perhaps use explicit quotes to show that the allowlist is empty in the above example?

Reply:
```suggestion
            webManager.logEvent("Updated Admin Console access configuration.", "Blocklist = {" + String.join(", ", blockedSet) +"}, Allowlist = {" + String.join(", ", allowedSet)+"}, Ignore excludes = {"+ignoreExcludes+"}");
```

How about this? Are braces too unweildy for this kind of logging?

![image](https://user-images.githubusercontent.com/2117083/182953189-2f0e4183-1412-4afd-92f2-307316fccfde.png)

Reply:
I think that the braces are a good idea. I have add those.

As for the newlines: I've seen them used in different places, as a kind of convention. The details don't show unless they're opened, so I think those newlines should simply be rendered properly. I've raised https://igniterealtime.atlassian.net/browse/OF-2503 for this.